libc-top-half/musl/src/internal/locale_impl.h

```diff
@@ -28,7 +28,7 @@ extern hidden const struct __locale_struct __c_dot_utf8_locale;
 hidden const struct __locale_map *__get_locale(int, const char *);
 hidden const char *__mo_lookup(const void *, size_t, const char *);
 hidden const char *__lctrans(const char *, const struct __locale_map *);
-hidden const char *__lctrans_cur(const char *);
+const char *__lctrans_cur(const char *);
```

Comment:
is this intended?

Reply:
Yes - libc-top-half/musl/src/string/strsignal.c (which is included in `LIBWASI_EMULATED_SIGNAL_MUSL_SOURCES`) uses `LCTRANS_CUR` (which is defined to be `__lctrans_cur`), which means libc.so must export it so libwasi-emulated.so can import it.

Reply:
I think this warrents and `#ifdef  __wasilibc_unmodified_upstream` along with a comment.

This is another reason why splitting libc into multiple libraries is maybe not the best idea.